src/test/java/MainTest.java

```diff
@@ -80,4 +82,17 @@ public static void main(String[] args) {
 
         Main.main(PORT);
     }
+
+    private static DbCredentials resolveDbCredentials() {
+        if (getOptionalProperty("lokal.database").isPresent()) {
```

Comment:
@jan-berge-ommedal 
Synes egentlig det er litt kjipt at default ved lokal kjøring er å bruke databasen i miljøet. Blir veldig fort rot av sånt. Hadde det ikke vært bedre om defaulten var å bruke in-memory hsql?

Reply:
Jeg snudde på flisa, sånn at lokal hsql blir default